### PR TITLE
Document sending custom tables alongside messages

### PR DESCRIPTION
**Pull request by severinraez**

### CODE REVIEW

File: custom_tables.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+* If no record is present it will be created.
+* If there already is a record, it will be updated with the fields you provide.
+* To delete an existing record, pass `{ "_delete": true }`
+  * This operation is idempotent, it will not fail if no record is present.
+  * `_delete` is not supported as a table column name.
+
+For 1:n tables, each array entry is handled as follows
+
+* If no id is present the record will be created.
+* If an id is present, the record will be updated with the fields you provide.
```
Comment (skaestle) [inline custom_tables.md:94]: will there be an error if the id does not exist?

File: custom_tables.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+* If only the keys `id` and `_delete` are present and `_delete` is `true`, the record with the given id will be deleted.
+  * `_delete` is not supported as a table column name.
+
+## Mapping of references
+
+Custom tables belong to a business entity. Furthermore, they might have FKs pointing to the ids of other business entities.
+On our APIs, we use the reference of business entities as their identifier. To allow you to fill FKs in custom tables if you only know the reference and not the id, the API
+will map given references to ids.
```
Comment (skaestle) [inline custom_tables.md:102]: newline not needed

File: custom_tables.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+So, given this 1:1 custom table `person_objekt_rating`
```
Comment (skaestle) [inline custom_tables.md:104]: isnt this an m:n mapping between person/objekt?

File: custom_tables.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+and an entry in `objekte` with reference `"000123.45.000678"` and id `23`,
+you can manage the table via the API as follows
+
+```json
+{
+    "data": {
+        "customTables": {
+            "person_objekt_rating": {
+                "rating": 5,
+                "objekt_reference": "000123.45.000678"
```
Comment (skaestle) [inline custom_tables.md:122]: will there be an error if referenz is not found?

Comment (skaestle) [reply custom_tables.md:122]: also, what is the person_id in this example?

**Review by skaestle — APPROVED** [review]
thanks for the doc!

File: custom_tables.md (hunk 1 above)
Comment (severinraez) [reply custom_tables.md:94]: yes, we delegate to the custom model commands and return all of their validations

File: custom_tables.md (hunk 3 above)
Comment (severinraez) [reply custom_tables.md:104]: technically yes, there's a definition of what i mean by 1:1 at the top of the document.
but this seems unclear... i'll change the wording to "1:1 _custom_ table" in the definition and everwhere. or do you have another idea?

File: custom_tables.md (hunk 4 above)
Comment (severinraez) [reply custom_tables.md:122]: there will be an error on invalid references.
what the person_id will be is explained above. i will add eventType and personReference fields here for clarification.

File: custom_tables.md (hunk 3 above)
Comment (skaestle) [reply custom_tables.md:103]: sounds like a plan

**Pull request merged**